org.eclipse.winery.topologymodeler.ui/TmDependencyTestApp/package.json

```diff
@@ -0,0 +1,68 @@
+{
+  "name": "tm-dependency-test-app",
+  "version": "0.0.0",
+  "license": "MIT",
+  "scripts": {
+    "ng": "ng",
+    "start": "ng serve",
+    "build": "ng build --prod",
+    "test": "ng test",
+    "lint": "ng lint",
+    "e2e": "ng e2e"
+  },
+  "private": true,
+  "dependencies": {
+    "@angular-redux/store": "6.5.7",
+    "@angular/animations": "6.0.4",
```

Comment:
In general: please ensure that you use the same versions as in the repository ui and the workflowmodeler.